packages/frontend/src/app/repository/repository.component.ts

```diff
@@ -32,43 +32,49 @@ import {RouteProvider} from '../routing/route-provider/RouteProvider';
   styleUrls: ['./repository.component.css'],
 })
 export class RepositoryComponent implements OnInit {
-  // mock builds until the data is fully formatted
-  mockBuilds = [
+  mockTests: Test[] = [
```

Comment:
Would these fields be enough? Currently I am not storing the environment for each build, but I could add it. I will add a new api endpoint to get the tests for the most recent build.

flaky: boolean
Name:String
Passed: boolean
percent passing: float (for last 30 builds)
status: "OK" or "error message

Reply:
LGTM !